app/static/settings_style.css

```diff
@@ -0,0 +1,149 @@
+/* Settings page */
+
+.sub-title {
+    font-size: 1rem;
+    padding-top: 0.2rem;
+}
+
+.settings {
+  color: #222831;
+  display: flex;
+  font-size: 0.9em;
+}
+
+#settings-left {
+  padding: 1.5em 1.5em;
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(230, 230, 230)
+}
+
+.settings-headline {
+  display: flex;
+  width: 200px;
+  padding-bottom: 1em;
+}
+
+.settings-layout {
+  display: flex;
+}
+
+.left-options-bar {
+  display: flex;
+}
+
+.settings-main article {
+  margin-bottom: 2em;
+}
+
+.settings-options {
+  flex: 1
+}
+
+.settings-options ul {
+  list-style-type: none;
+  padding-left: 0;
+  padding-right: 1em;
+}
+
+.settings-options ul li {
+  margin-bottom: 15px;
+  font-weight: bold;
+}
+
+.settings-options ul li a {
+  color: #222831;
+  transition: all .3s ease-in-out;
+  cursor: pointer;
+  text-decoration: none;
+}
+
+.settings-options ul li a:hover, .settings-options ul li a.active {
+  color: #5786f5;
+}
+
+.tabShow {
+  display: block;
+  transition: all .5s ease-in;
+}
+
+.tabHide {
+  display: none;
+}
+
+.settings-main {
+  flex: 3;
+  padding: 1.5em 1.5em;
+}
+
+.settings-main h2 {
+  margin-bottom: 1em;
+}
+
+.settings-main p {
+  margin: 1em 0em;
+}
+
+.settings-main .form-select {
+  width: 250px;
+}
+
+.tabShow p {
+  display: block;
+  align-self: left;
+}
+
+.tabShow h2 {
```

Comment:
names are lowercase and separated by "-".

Reply:
Thanks for the review! All typos fixed.